src/modules/Dashboard/features/RespondentData/RespondentDataReview/Feedback/FeedbackNotes/FeedbackNotes.hooks.ts

```diff
@@ -140,14 +103,10 @@ export const useFeedbackNotes = ({ entity: { id, isFlow } }: FeedbackNotesProps)
     const commonParams = { appletId, note: newNote };
 
     if (isFlow && submitId) {
-      createFlowNote({ ...commonParams, submitId, flowId: id });
-
-      return;
+      createNote({ ...commonParams, submitId, flowId: id });
```

Comment:
let's move the condition logic `if (isFlow && submitId) {...}` as a part of `createNote` function